flygym/fly.py

```diff
@@ -922,6 +887,64 @@ def _add_force_sensors(self):
 
         return force_sensors
 
+    def _add_joint_actuators(self, gain, forcerange):
+        # if self control is "motor" check that the gain is not provided
+        if self.control == "motor" and gain is not None:
+            # print warning
+            logging.warning(
+                "Motor control is selected, the gain parameter will not be used"
+            )
+
+        ## Need to deal with the kp, force range, and neck actuators
+        if not type(gain) == list:
+            gain = [gain] * len(self.actuated_joints)
+        if not type(forcerange) == list:
+            if type(forcerange) == tuple:
+                forcerange = [forcerange] * len(self.actuated_joints)
+            else:
+                forcerange = [(-forcerange, forcerange)] * len(self.actuated_joints)
+
+        actuators = []
+
+        for joint, g, forcerange in zip(self.actuated_joints, gain, forcerange):
+            if self.control == "position":
+                actuators.append(
+                    self.model.actuator.add(
+                        self.control,
+                        name=f"actuator_{self.control}_{joint}",
+                        joint=joint,
+                        kp=g,
+                        ctrlrange="-1000000 1000000",
+                        forcerange=forcerange,
+                        forcelimited=True,
+                    )
+                )
+            elif self.control == "velocity":
+                actuators.append(
+                    self.model.actuator.add(
+                        self.control,
+                        name=f"actuator_{self.control}_{joint}",
+                        joint=joint,
+                        kv=g,
+                        ctrlrange="-1000000 1000000",
+                        forcerange=forcerange,
+                        forcelimited=True,
+                    )
+                )
+            elif self.control == "motor":
+                actuators.append(
+                    self.model.actuator.add(
+                        self.control,
+                        name=f"actuator_{self.control}_{joint}",
+                        joint=joint,
+                        ctrlrange="-1000000 1000000",
+                        forcerange=forcerange,
+                        forcelimited=True,
+                    )
+                )
```

Comment:
I suggest adding an else clause:
```
else:
    raise ValueError(f"Invalid control type {control}.")
```
If something like this already exists in `__init__`, I suggest removing it and putting it here instead.